[css-anchor-position-1] Re-import css/css-anchor-position WPT

Scony opened the pull request:
#### 8f748e5a3e2c626ac38abdf37c84498fa9163289
<pre>
[css-anchor-position-1] Re-import css/css-anchor-position WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=281290">https://bugs.webkit.org/show_bug.cgi?id=281290</a>

Reviewed by Tim Nguyen.

This change re-imports css/css-anchor-position WPT test cases.
Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/671df7f85e35ae6400f530f6263e95458420d4aa">https://github.com/web-platform-tests/wpt/commit/671df7f85e35ae6400f530f6263e95458420d4aa</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-invalid-fallback-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-invalid-fallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-parse-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-parse-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-size-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-size-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/w3c-import.log:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/285249@main">https://commits.webkit.org/285249@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dd63ccdc6c50afccb6eea9a5e60baa1928aca95d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/71896 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/51316 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/24681 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/76056 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/23105 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/59117 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/22925 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/56747 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/15251 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/74963 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/46549 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/61942 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/37185 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/43212 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/19412 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/21450 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/65124 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/19776 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/77736 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/16136 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/18958 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/64900 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/16182 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/61965 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/64488 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/15908 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/12661 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/6303 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/47114 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/48183 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/49470 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/47927 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/TestExpectations:
```diff
@@ -8150,6 +8150,10 @@ imported/w3c/web-platform-tests/css/css-anchor-position/position-visibility-remo
 
+# crashes
+imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-004.html [ Crash ]
+imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children.html [ Crash ]
```
usually we just Skip crashes

Scony committed:
[css-anchor-position-1] Re-import css/css-anchor-position WPT
https://bugs.webkit.org/show_bug.cgi?id=281290

Reviewed by Tim Nguyen.

This change re-imports css/css-anchor-position WPT test cases.
Upstream commit: https://github.com/web-platform-tests/wpt/commit/671df7f85e35ae6400f530f6263e95458420d4aa

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-invalid-fallback-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-invalid-fallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-parse-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-parse-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-size-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-size-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/position-anchor-target-with-children.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/w3c-import.log:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-anchor-position/popover-implicit-anchor.tentative-expected.txt: Added.

Canonical link: https://commits.webkit.org/285249@main
